backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -430,6 +474,12 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 		opt:   *opt,
 		pacer: fs.NewPacer(pacer.NewGoogleDrive(pacer.MinSleep(minSleep))),
 		cache: bucket.NewCache(),
+		pool: pool.New(
+			time.Duration(opt.MemoryPoolFlushTime),
+			int(opt.ChunkSize),
+			fs.Config.Transfers,
```

Comment:
In S3 we use `opt.UploadConcurrency*fs.Config.Transfers,` which makes sense for a pool `poolSize`?

Reply:
GCS doesn't have `UploadConcurrency`, there are only `fs.Config.Transfers` in parallel.